src/components/MovieBooking/MovieSeats.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from "react";
+import MovieModal from "./MovieModal";
+import { useParams } from "react-router-dom";
+import SvgImg from '../Assets/SvgImg'
+import {
+  SeatStyle,
+  SeatTableStyle,
+  SeatButtonStyle,
+  ListStyle,
+  SeatRowStyle,
+  SeatListStyle,
+} from "../Styles/Seat.style";
+
+const seatsData = {
+  id: ["A", "B", "C", "D"],
+  seat: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
+};
+const MovieSeats = () => {
+  const [visible, setVisible] = useState(false);
+  const [selectSeats, setSelectSeats] = useState([]);
+  let param = useParams();
+
+  let propsData = {
+    selectSeatArray: selectSeats,
+    visibleFun: setVisible,
+    selectSeatFun: setSelectSeats,
+  };
+
+  let selectedSeat = [];
+  if (JSON.stringify(localStorage.getItem(param.id)) !== "null") {
+    selectedSeat = localStorage.getItem(param.id);
+  }
+
+  const ModalShow = () => {
+    if (selectSeats.length === 0 || selectSeats.length > 10) {
+      alert("please select the seats in range (1-10) ");
+    } else {
+      selectedSeat = selectedSeat + selectSeats;
+      localStorage.setItem(param.id, selectedSeat);
+      selectedSeat = localStorage.getItem(param.id);
+      setVisible(true);
+    }
+  };
+
+  const clickHandel = (id) => {
+    if (selectedSeat.includes(id)) {
+      alert("already selected");
```

Comment:
The seat is already selected please disable pointer events, instead of showing alert.